virocon/plotting.py

```diff
@@ -154,7 +254,29 @@ def plot_2D_isodensity(model, sample, model_desc=None, swap_axis=False, ax=None)
     return ax
 
 
+
 def plot_2D_contour(contour, sample=None, design_conditions=None, model_desc=None, swap_axis=False, ax=None):
+    """
+    Beschreibung 
+       
+    Parameters
+    ----------
+    contour: Contour
+        The environmental contour that should be plotted.
+    sample : ndarray of floats, optional
+        The environmental data sample that should be plotted against the fit.
+    design_conditions : array-like, optional
```

Comment:
array_like or boolean, optional
Specified environmental conditions under which the system must operate.
If an array it is assumed to be a 2D array of shape (number of points, 2) and should contain the precalulated design conditions. If it is True design_conditions are computed with default values and plotted. Otherwise no design conditions will be plotted (the default).